### DISCUSSION TITLE
Default selected country

### DISCUSSION THREAD

**Discussion by Faber44** [parfaitementweb/filament-country-field#13] [Q&A]
Hello!

I just want to select a default country.
I tried with that :
```
Country::make('country_iso')
    ->default('FR')
    ->selectablePlaceholder(false)
    ->label('Pays'),
```
But it doesn't work.

Any idea ?

**Comment by AlexisSerneels** [comment]
Are you trying to do so in the Create or Edit View?
Filament as this weird behavior where defaults values on works on Create. Otherwise, you should use ->fillForm().

**Reply by eugenefvdm** [reply]
Thanks! In my case on Create it works with ->default() and the code 'ZA' is case sensitive.
